packages/scorchwrap/README.md

```diff
@@ -0,0 +1,145 @@
+# ScorchWrap PoC
+
+> It's like stretchwrap, but with lava
+
+ScorchWrap is a webpack plugin that wraps each module in the bundle in a container and enforces LavaMoat Policies per package.
+
+Implemented capabilities:
+
+ - [x] wrapping built module sources 
+ - [x] runtime to make wrapped modules work
+ - [x] turn off concatenateModules and warn it's incompatible
+ - [x] inlining runtime into the bundle
+ - [x] including actual policies
+ - [ ] using policies to control module loading at runtime
+ - [ ] using policies to limit globals at runtime
+ - [ ] researching and covering all `__webpack_require__.*` functions for security as needed 
+ - [ ] support dynamic imports/requires reaching beyond the bundled content
+ - [ ] paranoid mode checks
+ - [ ] research potential concatenateModules support
+ - [ ] ecosystem compatibility quirks we're yet to find
+
+*(for more specific issues to solve see TODO comments in code)*
+
+
+
+## Usage
+
+The ScorchWrap plugin takes an optional options object with the following properties:
+
+- runChecks: Optional boolean property to indicate whether to check resulting code with wrapping for correctness. Default is false.
+- diagnosticsVerbosity: Optional number property to represent diagnostics output verbosity. A larger number means more overwhelming diagnostics output. Default is 0.  
+  Setting positive verbosity will enable runChecks.
+
+```js
+const ScorchWrapPlugin = require('@lavamoat/scorchwrap');
+
+module.exports = {
+  // ... other webpack configuration properties
+  plugins: [
+    new ScorchWrapPlugin({
+      policy: { ... LavaMoat policy structure ... }
+      runChecks: true,
```

Comment:
is this "paranoid" mode?  unclear if this has been implemented based on the language below

Reply:
not implemented yet. It's an idea for a set of features that may be less performant or theoretically unnecessary